server/sonar-web/src/main/js/apps/issues/sidebar/SeverityFacet.js

```diff
@@ -36,17 +37,18 @@ type Props = {|
   severities: Array<string>,
   stats?: { [string]: number }
 |};
+*/
 
 export default class SeverityFacet extends React.PureComponent {
-  props: Props;
+  /*:: props: Props; */
 
   static defaultProps = {
     open: true
   };
 
   property = 'severities';
 
-  handleItemClick = (itemValue: string) => {
+  handleItemClick = (itemValue /*: string */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)